### DISCUSSION TITLE
Count of repositories is 1 while I don't have any

### DISCUSSION THREAD

**Discussion by OlegShirochenkov** [community/community#47564] [Repositories]
### Select Topic Area

Bug

### Body

Hello.
I don't have any repositories, but GitHub shows me my count of repositories as 1.
I would submit this as a GitHub bug, but I don't have any paid support license.
Screenshot with no filters set as a proof is attached.
<img width="769" alt="Screenshot 2023-02-15 at 18 41 47" src="https://user-images.githubusercontent.com/49797227/219077177-36742642-a40b-4a48-a518-6c5d8519e369.png">

**Comment by radar-blip** [comment]
It's not showing that 1 repo from my side, maybe just a cache thing.

![xthis](https://user-images.githubusercontent.com/39196767/219099226-cce29f06-592e-4f2f-9766-815897f04ebc.png)

**Reply by OlegShirochenkov** [reply]
> It's not showing that 1 repo from my side, maybe just a cache thing.

Unfortunately it's not a cache, because I see the same on different devices for months or even years. I've just found a time to report this issue.

Seems this is related only to self-profile viewing.
